### PR TITLE
Fix readme links to asset library

### LINKED ISSUE #1077: fix all the links in the readme they are pointing to the old 3.5 versions of the demos
**Which demo project is affected:**

IDK probably all of them but at least two, I didn't check all of them

**OS/device including version:**

Windows 11, Godot 4.2.2 but it doesn't matter cuz I'm not actually running the projects 🤣 

**Issue description:**

i click on link in 4.2 version, but it takes me to 3.5 version in asset library

**Screenshots of issue:**

### PR DESCRIPTION
**Pull request by ShatteredRealityX**
update to latest version not 3.5

fixes #1077

could not find links to the following:
* 3D Inverse Kinematics
* Compositor Effects
* UI Mirroring
* 2.5D Demo Project with C#
* Android IAP with C#
* Dodge the Creeps with C#
* Pong Multiplayer with C#
* WebRTC Minimal
* WebRTC Signaling Demo

so i removed the links

squash the creeps screenshot readme removed
remove note about godot 3.2.3 in readme mono

### CODE REVIEW

**Comment by ShatteredRealityX** [comment]
oh right forgot to mention i also fixed all the renderers in the readme

File: 2d/README.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ These demos are all 2D, but otherwise do not have a common theme.
 Languages: Most have GDScript, some have
 [Godot shader language](https://docs.godotengine.org/en/latest/tutorials/shaders/shader_reference/shading_language.html)
 
-Renderers: 4 of them are GLES 3, but most are GLES 2
+Renderers: Glow for 2D and Physics Platformer use Foward+, 2D Particles uses Mobile, and the rest use Compatibility
```
Comment (Calinou) [inline 2d/README.md:8]:
```suggestion
Renderers: Glow for 2D and Physics Platformer use Forward+, 2D Particles uses Mobile, and the rest use Compatibility
```

File: 3d/README.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ These demos are all 3D, but otherwise do not have a common theme.
 
 Languages: All are GDScript
 
-Renderers: Truck Town, Physics Tests, and Waypoints are GLES 2, the rest are GLES 3
+Renderers: Truck Town uses Mobile on mobile platforms, the rest use Forward+
```
Comment (Calinou) [inline 3d/README.md:7]: All demos use Mobile on mobile platforms automatically (it's the default project setting override), so I wouldn't mention it explicitly:

```suggestion
Renderers: Forward+
```

File: 2d/README.md (hunk 1 above)
Comment (ShatteredRealityX) [reply 2d/README.md:8]: I have been misspelling that word since third grade 😭

**Comment by ShatteredRealityX** [comment]
i fixed it but idk how to squash it without github giving me some kind of merging error

**Comment by AThousandShips** [comment]
You need to push with `git push --force`

File: 2d/README.md [hunk 3]
```diff
@@ -5,4 +5,4 @@ These demos are all 2D, but otherwise do not have a common theme.
 Languages: Most have GDScript, some have
 [Godot shader language](https://docs.godotengine.org/en/latest/tutorials/shaders/shader_reference/shading_language.html)
 
-Renderers: 4 of them are GLES 3, but most are GLES 2
+Renderers: Glow for 2D and Physics Platformer use Forward+, 2D Particles uses Mobile, and the rest use Compatibility
```
Comment (AThousandShips) [inline 2d/README.md:8]:
```suggestion
Renderers: Glow for 2D and Physics Platformer use Forward+, 2D Particles uses Mobile, and the rest use Compatibility

```

**Pull request merged**